### DISCUSSION TITLE
About oneway MTB track and oneway:bicycle

### DISCUSSION THREAD

**Discussion by FrankWessels** [waymarkedtrails/waymarked-trails-site#402] [Q&A] [answered]
Hello all,

My question is:

is it true that of the route has property "oneway": "yes" and if at least one of each members has "oneway": "yes" and if all members that have "oneway": "yes" are consistent, then directional arrows are shown on the map?

If yes, is it possible to make it in such way that it can also use property "oneway:bicycle": "yes" of one of it's members?

If no, what exactly triggers that arrows are shown on the map? I would like to adjust all routes I cycle myself such that these arrows become visible?

Regards,
Frank

**Comment by lonvia** [comment, marked as answer]
waymarkedtrails.org doesn't currently process or show 'oneway' on MTB routes. The underlying streets and paths show an arrow when the ways have a oneway tag. But that is on the basemap whose rendering style is outside the control of the waymarkedtrails website. Please don't add oneway tags simply to make arrows render on the map.

Support for oneway routes is somewhere on the TODO list but I cannot promise anything, when it will be implemented.

**Comment by FrankWessels** [comment]
Hi Sarah,

Ah, clear. Don't worry, I am adding oneway or oneway:bicycle tags first of all because it is correct extra information.

Indeed the basic problem that I have is that some oneway MTB ways are also open for pedestrians. So the oneway tag doesn't apply because pedestrians can go both ways. So it makes sense that in this case arrows are not rendered on the normal map, even if it has the tag oneway:bicycle.

However, when trying to plan a nice MTB route linking multiple routes together, there isn't yet a MTB/cycling map/software which shows the oneway:bicycle information. And I would like to avoid accidentally going the opposite way, which can lead to nasty incidents. So now I simple look at the map details of openstreetmap directly which is quite the effort.

I think if you are ever going to add it, it is already super helpful to simply render arrays on ways with the tag oneway:bicycle (when in MTB/bicycle mode). It would be much harder to implement (with at least for me only a bit of additional value) to add some logic to render complete MTB loops with arrows if the MTB route is oneway and at least one member gives direction and everything is consistent.

Thanks for your information,

Regards,
Frank
